packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/adding-basic-auth/index.md

```diff
@@ -2,20 +2,24 @@
 title: Adding Basic Authorization and Body Parsing
 ---
 
-The Glitch projects do not have any authorization or body parsing code by default. To include this content:
+The default, new Glitch projects do not have any authorization or body parsing code. To include this content:
 
 * Add the Body Parser and Basic Auth `npm` packages to your Glitch project
 * Add the code snippet below
 
+If you copied a Glitch project (re-mixed it), the packages and code are already included.
+
 To add the `npm` packages:
 
 1. Click on the `package.json` file in the left-hand project menu.
 2. From the `Add Package` drop-down, search for the `express-basic-auth` and `body-parser` packages.
 3. Click each package to add to your project.
 
-After including the packages, add the following code snippet in your project.
+The Inline Hook examples use [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication) to authenticate the Okta Inline Hook API calls by your Glitch external service. The user and password credential pair, stored in your Glitch application, must be encoded in Base64 and added as the Authentication secret when you activate the Inline Hook on your Okta org. Ensure you add the scheme `Basic` as a prefix to the authentication secret value.
```

Comment:
suggestions:
The Inline Hook examples use [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication) to authenticate the Okta Inline Hook API calls **made by your external Glitch** service. The **username** and password credential pair stored in your Glitch application must be encoded in **Base64. Add them as the authentication secret** when you activate the Inline Hook on your Okta org. Ensure you add the scheme `Basic` as a prefix to the authentication secret value.

Reply:
I rushed the re-write on this paragraph and obviously didn't do too well. The external service (Glitch application) authenticates the Okta Inline Hook API call using Basic Auth. (The authentication is by or at the external service, not the calls made by the service are authenticated, which I can totally see now is what the opening appears to be stating.)

username instead of user is correct, thanks

The username:password pair are not encoded in Base64 on the external service; they're encoded as part of the API request header, which is added on Okta side via the Inline Hook config/activation process using the "Authentication secret" (UI field).

Let me take another crack at this paragraph.

Reply:
Revised as follows:

The Inline Hook examples use [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication) to authenticate the Okta Inline Hook API calls received by your Glitch external service. In the Okta org, the Glitch project username and password credentials must be encoded in Base64 and added as the Authentication secret when you activate the Inline Hook. Ensure you add the scheme `Basic ` (including a space) as a prefix to the authentication secret value.